src/sql/media/icons/common-icons.css

```diff
@@ -308,23 +308,22 @@
 	background-image: url('stop_cell_solidanimation_inverse.svg');
 }
 
-.codicon.arrow-up {
-	background-image: url("chevron_up.svg");
-}
-.vs-dark .codicon.arrow-up,
-.hc-black .codicon.arrow-up {
-	background-image: url("chevron_up_inverse.svg");
-}
 
-.codicon.arrow-down {
-	background-image: url("chevron_down.svg");
-}
-.vs-dark .codicon.arrow-down,
-.hc-black .codicon.arrow-down {
-	background-image: url("chevron_down_inverse.svg");
+
+/* Icons as masked elements for easy theme switching.
+Includes non-masked style declarations. */
+.masked {
```

Comment:
Can we just remove this bit until we figure out how we want to handle the different supported themes? I'm not a fan of adding in temporary code like this - especially when it's so vastly different than how we do icons everywhere else.

(if we do decide to go with though putting a note here that we need to update the selector - masked is too generic of a class IMO given what it's doing. This seems very specific to icons so I'd suggest a class name of masked-icon)

Reply:
This is crucial to the presentation of markdown toolbar icons, and anywhere else I have used masked icons. I can rename this class to what you suggest, then I need to update my elements in the component.

Reply:
Yeah so given our discussion I'm fine with leaving the masking stuff in but it'd be great if you could update the name.

Reply:
Class updated to `masked-icon`